testing/simapp/simd/cmd/root.go

```diff
@@ -205,7 +205,6 @@ func initRootCmd(rootCmd *cobra.Command, encodingConfig params.EncodingConfig, b
 
 	// add keybase, auxiliary RPC, query, genesis, and tx child commands
 	rootCmd.AddCommand(
-		rpc.StatusCommand(),
```

Comment:
I see that [SDK v0.50-rc0 is doing here `server.StatusCommand()`](https://github.com/cosmos/cosmos-sdk/blob/v0.50.0-rc.0/simapp/simd/cmd/commands.go#L129). Should we do the same?

Reply:
ahhh, thank you for referencing upstream -- I'd wrongly assumed that it had joined the bundle of commands.